driver/src/main/scala/com/stratio/sparkta/driver/service/StreamingContextService.scala

```diff
@@ -87,6 +87,7 @@ case class StreamingContextService(policyStatusActor: Option[ActorRef] = None, g
           synchronized {
             log.info("Stopping message received from Zookeeper")
             SparkContextFactory.destroySparkStreamingContext
+            SparkContextFactory.destroySparkContext
             policyStatusActor.get ? Update(PolicyStatusModel(policyId, PolicyStatusEnum.Stopped))
             nodeCache.close()
             if (exit) {
```

Comment:
I think that we must remove the line after the if(exit):

              SparkContextFactory.destroySparkContext

Now is not necessary